DogeNet.IdentityServer/Styles/app.css

```diff
@@ -0,0 +1,173 @@
+﻿.gradient-button {
+    text-decoration: none;
+    display: inline-block;
+    color: white;
+    width: 100%;
+    height: 40px;
+    margin-bottom: 5px;
+    border-radius: 20px;
+    font-family: "Montserrat", sans-serif;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    background-image: linear-gradient( to right, #9eefe1 0%, #4830f0 51%, #9eefe1 100% );
+    background-size: 200% auto;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.1);
+    transition: 0.5s;
+}
+
+.gradient-button:hover {
+    background-position: right center;
+}
+
+.errorMsg {
+    color: red;
+}
+
+input {
+    outline: none;
+    border: none;
+}
+
+textarea {
+    outline: none;
+    border: none;
+} 
+
+
+button {
+    outline: none !important;
+    border: none;
+    background: 0 0;
+}
+
+button:hover {
+    cursor: pointer;
+}
+
+
```

Comment:
Empty lines? Code checker for css files?

Reply:
fixed